### DISCUSSION TITLE
Cloud backup using Google Drive or Dropbox

### DISCUSSION THREAD

**Discussion by PyrokineticDarkElf** [anonaddy/anonaddy#596] [Feature Requests]
I read a request for an automatic backup that would email your aliases to you upon change.

This is a 1-up on that request... Could we have an automatic/periodic backup of our data saved to Google Drive or Dropbox?

🙏

**Comment by DeandrePetersen** [comment]
Your proposal for automatic cloud backups to Google Drive or Dropbox is intriguing and could significantly enhance data security and accessibility. It's essential to explore various options for safeguarding data comprehensively. 
While Google Drive and Dropbox offer convenient storage solutions, have you considered integrating [Office 365 cloud backup](https://loopbackup.com/cloud-to-cloud-backup-for-microsoft-office-365/) into your strategy? Solutions like LoopBackup specialize in protecting Microsoft Office 365 data, providing robust backup capabilities tailored to your specific needs. 
By diversifying your backup approach, you can ensure resilience against potential data loss scenarios and enjoy greater peace of mind.
